src/components/WindowSideBarButtons.js

```diff
@@ -39,49 +46,59 @@ export class WindowSideBarButtons extends Component {
    * @return {type}  description
    */
   render() {
-    const { hasAnnotations, t } = this.props;
+    const {
+      classes, hasAnnotations, sideBarPanel, t,
+    } = this.props;
+
     return (
-      <>
-        <IconButton
+      <Tabs
+        classes={{ flexContainer: classes.tabsFlexContainer, indicator: classes.tabsIndicator }}
+        value={sideBarPanel === 'closed' ? false : sideBarPanel}
+        onChange={this.handleChange}
+        variant="fullWidth"
+        indicatorColor="secondary"
+        textColor="secondary"
+        aria-orientation="vertical"
+      >
+        <Tab
+          classes={{ root: classes.tab, selected: classes.tabSelected }}
           aria-label={
             this.sideBarPanelCurrentlySelected('info')
-              ? t('closeInfoCompanionWindow')
+              ? t('closeCompanionWindow')
               : t('openInfoCompanionWindow')
           }
-          onClick={() => (this.toggleCompanionWindow('info'))}
-        >
-          <InfoIcon
-            color={this.sideBarPanelCurrentlySelected('info') ? 'secondary' : 'inherit'}
-          />
-        </IconButton>
-        <IconButton
+          icon={(
+            <InfoIcon />
+          )}
+          value="info"
+        />
+        <Tab
+          classes={{ root: classes.tab, selected: classes.tabSelected }}
           aria-label={
             this.sideBarPanelCurrentlySelected('canvas_navigation')
               ? t('closeCanvasNavigationCompanionWindow')
               : t('openCanvasNavigationCompanionWindow')
           }
-          onClick={() => (this.toggleCompanionWindow('canvas_navigation'))}
-        >
-          <CanvasIndexIcon
-            color={this.sideBarPanelCurrentlySelected('canvas_navigation') ? 'secondary' : 'inherit'}
```

Comment:
Nice! We're able to remove this bit of logic because `Tab` has its own set of classes that define CSS for selected icons. This is set below in the `tabSelected` class in the WIndowSideBarButtons container.